solutions/two_sum.py

```diff
@@ -0,0 +1,36 @@
+"""
+This module contains a function to solve the two-sum problem.
+"""
+
+from typing import List
+
```

Comment:
A custom exception could provide a clearer indication of the issue when the input is less tan two numbers
 ```suggestion
class NoTwoSumSolutionError(Exception):
    """Raised when no two numbers in the list sum to the target."""
    pass

```